lib/circuitbox/timer/null_timer.rb

```diff
@@ -0,0 +1,9 @@
+class NullTimer
+  def initialize(time_unit = :milliseconds)
```

Comment:
i'd probably move this to the monotonic timer because it's not used in null timer or simple timer. Then I don't think anything would need to inherit from NullTimer